mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -364,3 +394,14 @@ class _SelfValidateJob implements HasStageName {
 
   _SelfValidateJob(this.stageName);
 }
+
+class _MapEntryWithStage<K, V> implements MapEntry<K, V> {
```

Comment:
@jakemac53  Remember – you can drop the type params here – we know what they are!

Reply:
See my other comment (kind of lost now) but I prefer to keep them because otherwise you can't see the K/V types where this type is used. There isn't any downside really afaik?

Reply:
🤷 – not a biggy, I guess.

It makes things cleaner, I think